Makefile

```diff
@@ -0,0 +1,91 @@
+
+VERSION := 0.1.0
+
+# Repository
+SRC_DIR := src
+
+PARSER_REPO_URL   ?= $(shell git -C $(SRC_DIR) remote get-url origin)
+$(eval PARSER_NAME=$(shell basename $(PARSER_REPO_URL) | cut -d '-' -f3 | sed 's#.git##' ))
+UPPER_PARSER_NAME := $(shell echo $(PARSER_NAME) | tr a-z A-Z)
+
+# install directory layout
+PREFIX ?= /usr/local
+INCLUDEDIR ?= $(PREFIX)/include
+LIBDIR ?= $(PREFIX)/lib
+PCLIBDIR ?= $(LIBDIR)/pkgconfig
+
+# collect C++ sources, and link if necessary
+CPPSRC := $(wildcard $(SRC_DIR)/*.cc)
+
+ifeq (, $(CPPSRC))
+	ADDITIONALLIBS := 
+else
+	ADDITIONALLIBS := -lc++
+endif
```

Comment:
In the empty case, this results in the command:
```
cc  -shared -Wl,,-soname,libtree-sitter-swift.so.0 src/parser.o src/scanner.o  -o libtree-sitter-swift.so.0.0
```

Note the two consecutive commas, which cause the linker to search for a library with the empty string as a name. This causes an error:
```
/usr/bin/ld: cannot find : No such file or directory
```

I was able to fix this locally by subsuming the second comma into `ADDITIONALLIBS`, so that in the `else` case, it's written as `-lc++,`. Then the `LINKSHARED` line only assumes it is responsible for a single comma.

Reply:
I see. that will fail the `tree-sitter.pc.in` on install. Let me come up with some fix to that.

Reply:
Alternately, since I don't anticipate adding any C++ source, we could probably drop this entirely.